lib/core/bytes.nit

```diff
@@ -538,6 +538,12 @@ class Bytes
 		length += cln
 	end
 
+	redef fun has(c)
+	do
+		if not c isa Int then return false
```

Comment:
While this makes sense given the current signature, I'm wondering why we need this.

What was the reason for `has.*` functions in `Collection[E]` to not be bound to `nullable E`?

Reply:
Bytes is an Abstract Array[Int], it’s make sense to return false when the type is not an Int for remove the list iteration. In the same case this comparison remove the type error for the int operation.

Reply:
The reason is basically to be consistent with `fun ==(nullable Object)`

note it also reduces the number of dynamic type errors: better return false than aborting.

~~~nit
var a: Array[Object] = [1,2,3]
print a.has("toto")
# there is obviously no toto, I will tell you, I won't crash your program for that
~~~